ACMILabs/media-player
# #156 Add current playlist position to sync messages

Resolves #156

Add current playlist position to sync messages so that clients can be sure they're playing the correct playlist item when syncing.

### Acceptance Criteria
- [x] Set `DEBUG` to `true` to see sync output
- [x] Investigate why they're not syncing
- [x] Add `current_playlist_position` to sync messages and set client to the correct playlist item when needed

### Relevant design files
* None

### Testing instructions
1. Setup a local Playlist with 3x 10 second videos in it (✔️manually trimmed with ffmpeg)
2. Setup two Raspberry Pi Media Players to be synced players - (already setup here: https://dashboard.balena-cloud.com/apps/1505457/devices)
3. Try playing them on `main` branch for a while to see if the error case occurs: `git checkout main` and `balena push s__media-player-pi-4`
4. Switch to this branch to see that its solved and all players stay in sync on the correct playlist item: `git checkout `fix/156-synced-playlists` and `balena push s__media-player-pi-4 `

### DoD
For requester to complete:
~- [ ] Deployment / migration instruction have been updated if required~
